index.html

```diff
@@ -2580,6 +2624,11 @@ <h3>
 that decision.
         </p>
 
+        <p>
+          Consumers SHOULD drop all properties from a DID Documents that 
+          are not defined by the <code>@context</code>. 
```

Comment:
```suggestion
          are not defined via the <code>@context</code>. 
```

`by` indicates that `@context` defines it directly... `via` indicates that it is defined "by way of" the `@context` ... that is, linking elsewhere is just as good as defining it inline. Might be too pedantic, but we should be consistent throughout the document.